src/forms/FormFieldAdvancedSelection/AdvancedSelection.test.tsx

```diff
@@ -252,7 +255,9 @@ describe("AdvancedSelection component", () => {
 		const inputNode = screen.getByPlaceholderText("Search...");
 		fireEvent.change(inputNode, { target: { value: "abc" } });
 
-		expect(await screen.queryByText("Option 1")).toBe(null);
+		await waitFor(() => {

```

Comment:
This test is still failing because the following warning is being thrown:

![image](https://user-images.githubusercontent.com/87880265/179779694-5084f095-8136-4b42-a35c-2695115b4dff.png)

You could make sure that the is passing if you run the following command: `yarn run test -- nameOfFile.test.tsx
` this will run only the tests related to the file that you specify thus, you save the time spent when all the tests are run.